src/cli.js

```diff
@@ -18,7 +18,6 @@ try {
     .arguments('[module]', 'prints module version from the node_modules')
     .option('-d, --debug', 'see full error messages, mostly for debugging')
     .option('-g, --generate-gif', 'should gif be generated')
```

Comment:
`-g` usually means `global`, not the best name for CLI option

Reply:
```-gif```
?